cmd/system/go.go

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) arkade author(s) 2022. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package system
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/alexellis/arkade/pkg/archive"
+	"github.com/alexellis/arkade/pkg/env"
+	"github.com/alexellis/arkade/pkg/get"
+	"github.com/spf13/cobra"
+)
+
+func MakeInstallGo() *cobra.Command {
+
+	command := &cobra.Command{
+		Use:   "go",
+		Short: "Install Go",
+		Long:  `Install Go programming language and SDK.`,
+		Example: `  arkade system install go
+  arkade system install go --version v1.18.1`,
+		SilenceUsage: true,
+	}
+
+	command.Flags().StringP("version", "v", "", "The version for Go, or leave blank for pinned version")
+	command.Flags().String("path", "/usr/local/", "Installation path, where a go subfolder will be created")
+	command.Flags().Bool("progress", true, "Show download progress")
+
+	command.PreRunE = func(cmd *cobra.Command, args []string) error {
+
+		return nil
+	}
+
+	command.RunE = func(cmd *cobra.Command, args []string) error {
+		installPath, _ := cmd.Flags().GetString("path")
+		version, _ := cmd.Flags().GetString("version")
+		fmt.Printf("Installing Go to %s\n", installPath)
```

Comment:
Should we introduce `--quiet` flag as we did earlier for binary downloads?

Reply:
I'm going to keep things verbose for the preview stages, to make it simpler / less work for everyone.

Then we can come back and fine tune.